src/components/work/Footer/Footer.tsx

```diff
@@ -14,20 +14,35 @@ import {
 } from 'reactstrap';
 import { Link } from 'react-router-dom';
 
+import { SETTINGS } from 'configs/settings';
 import { BeenestSVGPrimary } from 'shared/svgComponents/SvgComponents';
 
+const { BEENEST_HOST } = SETTINGS;
```

Comment:
Is `BEENEST_HOST` needed here? (Could the links just be to `/whatever` and let `BeeLink` or the browser do the rest?)

Reply:
using `BEENEST_HOST` to have it work between dev/staging/prod and `'/'` and `'/work'`

Reply:
main issue is that we have to use an `a` tag because of the html switching